src/vs/workbench/contrib/testing/browser/theme.ts

```diff
@@ -190,6 +190,15 @@ export const testStatesToIconColors: { [K in TestResultState]?: string } = {
 	[TestResultState.Skipped]: testingColorIconSkipped,
 };
 
+export const testStatesToRetiredIconColors: { [K in TestResultState]?: string } = {
+	[TestResultState.Errored]: testingColorIconErrored,
```

Comment:
These colors should be different than the non-retired colors, and customizable via `registerColor`'s for each of them

Reply:
I get that. The retired and non-retired colors should be different objects like `testingRetiredColorIconErrored`, `testingRetiredColorIconPassed`, etc. But I'm not sure what their color value should be.

Reply:
Their defaults can be `transparent(<original color>, 0.7)` to match the previous CSS